### PR TITLE
"Hw5 task2"

### PR DESCRIPTION
**Pull request by E1-Bart0**

### CODE REVIEW

File: test/hw5/test_task2.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from All_home_works.hw5.task2 import print_result
+
+
+def foo():
+    """DOCS OF FOO"""
```
Comment (Cjkjvfnby) [inline test/hw5/test_task2.py]: Please follow docstring convention.

File: test/hw5/test_task2.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    return "return foo"
+
+
+def test_doc_string():
+
+    foo_with_decorator = print_result(foo)
+    foo_with_decorator()
```
Comment (Cjkjvfnby) [inline test/hw5/test_task2.py]: Do you need to call it?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: test/hw5/test_task2.py (hunk 2 above)
Comment (E1-Bart0) [reply test/hw5/test_task2.py]: I made mistake. I appropriated doc and name before the function had been ever called which explains why it worked only after called.

Comment (Cjkjvfnby) [reply test/hw5/test_task2.py]: > which explains why it worked only after called

It's worth exploring such strange cases before requesting a review.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
I think we could add one more test.

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**